builtin/game/register.lua

```diff
@@ -418,6 +418,9 @@ function core.override_item(name, redefinition)
 	for k, v in pairs(redefinition) do
 		rawset(item, k, v)
 	end
+	for _, field in ipairs(del_fields) do
```

Comment:
needs to be optional

Reply:
Oh yeah, that'd cause a crash if ran without

Run a type(del_fields) == "table" ?

If you ran with a if del_fields then wouldn't that cause an error if it was given say, a number or boolean? ;p